scripts/base/init-bare.zeek

```diff
@@ -5203,6 +5203,32 @@ export {
 	## Number of bytes per packet to capture from live interfaces.
 	const snaplen = 9216 &redef;
 
+	## Default timeout for packet sources without file descriptors.
```

Comment:
I'm not overly happy this being in the Pcap module (as it's more generic to PktSrc), but given packet sources may just override `GetNextTimeout()` and ignore it, seems fair game, too.